hapi-base/src/main/java/ca/uhn/hl7v2/app/ActiveConnection.java

```diff
@@ -287,4 +287,8 @@ public boolean acceptAllMessages(){
 		return acceptAll;
 	}
 
+	// visibleForTesting
```

Comment:
I thought that annotation was availble in hapi too

Reply:
Not in Hl7V2